cw_bitcoin/lib/litecoin_wallet.dart

```diff
@@ -758,6 +758,13 @@ abstract class LitecoinWalletBase extends ElectrumWallet with Store {
     if (!mwebEnabled) return false;
     if (!tx.isPending) return false;
 
+    final isMwebTx = (tx.inputAddresses?.any((addr) => addr.contains("mweb")) ?? false) ||
+        (tx.outputAddresses?.any((addr) => addr.contains("mweb")) ?? false);
+
+    if (!isMwebTx) {
+      return false;
+    }
```

Comment:
well, since this is called only in 1 mweb-related function, shouldn't it initially ignore non-mweb transactions?
minor but we can look into it later to improve the code readability and maintainability